test/multiple-element-interactions-test.js

```diff
@@ -42,53 +44,55 @@ describe("When multiple DOM elements are present", () => {
             });
         });
 
-        it("can read attributes from custom child element's prototypes", () => {
-            var DataSource = components.newElement();
-            DataSource.data = [1, 2, 3];
-            components.registerElement("data-source", { prototype: DataSource });
+        // Pending until we decide on a good solution
```

Comment:
The test needn't be disabled because of that though. It'd be nice to be able to write it more cleanly, but the behaviour will be the same, and testing it like this is still important to check that child nodes do correctly get initialised and become accessible.